api/routes.js

```diff
@@ -4,7 +4,7 @@ const validate = require("./validate");
 
 const data = require("./db/data.js");
 
-//GET endpoint
+//GET endpoint for suggestions
 router.get("/:handle", async (req, res) => {
```

Comment:
Can use regex to avoid encoding the slash:
"/:handle([0-9]+.[0-9]+.[0-9]+/[0-9]+)"